### PR TITLE
Properties: Change "OK" button to Accent button style

### LINKED ISSUE #3130: Change "OK" action in properties dialog to Accent style
**Is your feature request related to a problem? Please describe.**
Currently in the properties dialog, there are two options: "OK" and "Cancel". These both use reveal style, however I think that it would both look better and improve the user experience by distinguishing these two buttons better.

**Describe the solution you'd like**
Change "OK" button in properties dialog to `AccentButtonStyle` to highlight the primary action of the dialog.

**Describe alternatives you've considered**
None

**Additional context**
image

### PR DESCRIPTION
**Pull request by XPoppyX**
Closes #3130 

- ----------

button1
button2

### CODE REVIEW

File: Files/Views/Pages/Properties.xaml [hunk 1]
```diff
@@ -99,7 +99,7 @@
                 HorizontalAlignment="Right"
                 Click="OKButton_Click"
                 Content="OK"
-                Style="{ThemeResource ButtonRevealStyle}" />
+                Style="{ThemeResource AccentButtonStyle}" />
```
Comment (tsvietOK) [inline Files/Views/Pages/Properties.xaml:102]: You can use `{StaticResource AccentButtonStyle}` here.

Comment (XPoppyX) [reply Files/Views/Pages/Properties.xaml:102]: ok

**Comment by mdtauk** [comment]
The accent button style is traditionally used for "default actions", so the action that will be performed when you press Enter, because that item has focus.

In modern UI, it is also used to distinguish destructive actions, or actions that will change something.

So its important to not only change the style, but to examine the behaviour of the button too.

File: Files/Views/Pages/Properties.xaml (hunk 1 above)
Comment (tsvietOK) [reply Files/Views/Pages/Properties.xaml]: In that case `AccentButton.xaml` and `packages.config` no more needed.

Comment (XPoppyX) [reply Files/Views/Pages/Properties.xaml]: removed

**Review by tsvietOK — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@XPoppyX Can you remove the reveal style from the cancel button as well to make it consistent with the "OK" button?

**Review by yaichenbaum — APPROVED** [review]

**Review by tsvietOK — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@XPoppyX Great work!

**Pull request merged**